docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture designed by Arc that enables it to use the reusable contracts provided by Arc.  Each DAO has an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
```

Comment:
> **The avatar**

Can we omit "The" ?

Reply:
You end up with more redundancy then:

**The avatar** is a public-facing

becomes:

**Avatar** The avatar is a public-facing

Reply:
How about : Avatar is a public-facing.